measures/HPXMLtoOpenStudio/measure.rb

```diff
@@ -2975,6 +2979,8 @@ def self.add_building_output_variables(runner, model, map_tsv_dir)
 
     dhw_output_vars = [OutputVars.WaterHeatingElectricity,
                        OutputVars.WaterHeatingElectricityRecircPump,
+                       OutputVars.WaterHeatingHeatExchanger,
+                       OutputVars.WaterHeatingIndirectBoiler,
```

Comment:
I suggest `WaterHeatingCombiBoilerHeatExchanger` and `WaterHeatingCombiBoiler`. The HX variable is only used for combi boilers, so lets make it more specific, and the indirect variable is also used for tankless coil, so lets make it more generic.